### PR TITLE
KR 5.35 update

### PR DESCRIPTION
**Pull request by Akurosia**

### CODE REVIEW

File: ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js [hunk 1]
```diff
@@ -1113,5 +1142,84 @@ export default {
...
+        'Surface Missile': '대지 미사일',
+        'Swooping Frenzy': '광란의 급강하',
+        'Taste Of Blood': '귀곡혈산참',
+        'The King\'s Notice': '패왕사시안',
+        'Thunder IV': '선더쟈',
+        'Tornado': '토네이도',
+        'Twin Agonies': '쌍마사왕참',
+        'Voltstream': '번개 급류',
+        'Warped Light': '섬광포',
+        'Water IV': 'ウォタジャ',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js:1220]:
```suggestion
        'Water IV': '워터쟈',
```

Comment (Akurosia) [reply ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js:1220]: Thanks for spotting. The ID of Water IV is '53D8' which is all japanese except for the english translation.

File: ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js [hunk 2]
```diff
@@ -1113,5 +1142,84 @@ export default {
...
+        'Eaglesight': '독수리 광장',
+        'Lightsphere': '빛 구체',
+        'Lyon The Beast King': '마수왕 라이언',
+        'Majesty\'s Auspice': '단상',
+        'Shadowsphere': '그림자 구체',
+        'The airship landing': '골드 소서 비공정 승강장',
+        'The grand gates': '성문',
+        'Verdant Plume': '진녹색 날개',
+      },
+      'replaceText': {
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js:1163]:
```suggestion
      'replaceText': {
        '--Lyon Passage--': '--라이언 통과--',
```
I don't exactly know about this, I translated this guessing this means that Lyon pass through something.

File: ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js [hunk 3]
```diff
@@ -1113,5 +1142,86 @@ export default {
...
+        'Command: Infrared Blast': '지령: 대물 열선',
+        'Command: Joint Attack': '지령: 대물 집중 공격',
+        'Command: Lateral Dive': '지령: 대물 돌진 공격',
+        'Command: Suppressive Formation': '대인 제압 돌격',
+        'Curse Of The Fiend': '마법인',
+        'Electric Anvil': '전기 모루',
+        'Energy Generation': '에너지 구체 생성',
+        'Explosion': '폭산',
+        'False Thunder': '인공 번개',
+        'Fervid Pulse': '열렬한 고동',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js:1186]:
```suggestion
        'Frigid(?! )': '냉랭한',
        'Fervid Pulse': '열렬한 고동',
```

File: ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js [hunk 4]
```diff
@@ -1113,5 +1142,86 @@ export default {
         'Winds\' Peak': '超级烈风波',
       },
     },
+    {
+      'locale': 'ko',
+      'missingTranslations': true,
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/data/05-shb/eureka/bozjan_southern_front.js:1147]:
```suggestion
```

**Review by Jaehyuk-Lee — COMMENTED** [review]
This finishes translation.

**Pull request merged**